src/org/usfirst/frc4904/robot/humaninterface/operators/BillyOP.java

```diff
@@ -0,0 +1,86 @@
+package org.usfirst.frc4904.robot.humaninterface.operators;
+
+
+import java.util.function.Supplier;
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.BallioCycle;
+import org.usfirst.frc4904.robot.commands.BallioFloorClear;
+import org.usfirst.frc4904.robot.commands.BallioIntake;
+import org.usfirst.frc4904.robot.commands.BallioOuttake;
+import org.usfirst.frc4904.robot.commands.FlywheelSpinup;
+import org.usfirst.frc4904.robot.commands.GearioIntake;
+import org.usfirst.frc4904.robot.commands.GearioOuttake;
+import org.usfirst.frc4904.robot.commands.SetOverride;
+import org.usfirst.frc4904.robot.commands.SetRampState;
+import org.usfirst.frc4904.robot.commands.Shoot;
+import org.usfirst.frc4904.robot.subsystems.GearIO;
+import org.usfirst.frc4904.standard.humaninput.Operator;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class BillyOP extends Operator {
+	public static final double INTAKE_THRESHOLD = 0.5;
+
+	public BillyOP() {
+		super("DefaultOperator");
```

Comment:
Make sure to change this to the name of your class so it displays properly on SmartDashboard!